tests/unit/test_resample.py

```diff
@@ -598,3 +602,248 @@ def test_get_bounds_var(var_info_fxn, test_IMERG_ncfile):
 
     actual_bounds = get_bounds_var(var_info, '/Grid/lon')
     assert expected_bounds == actual_bounds
+
+
+@pytest.mark.parametrize(
```

Comment:
I'm not sure this made it clearer.

Reply:
I can revert this commit because I wasn't sure this was better.

Reply:
[`7390a62` (#27)](https://github.com/nasa/harmony-regridding-service/pull/27/commits/7390a6254bbfe027b48779b97ba5944d20a44a2e)

Reply:
now back to 745dae4